HighwayPatrol.h

```diff
@@ -2,10 +2,17 @@
 
 #include "Vehicle.h"
 
-struct HighwayPatrol : Vehicle 
+struct Highway;
+
+struct HighwayPatrol : Vehicle
 {
     HighwayPatrol();
+
+    virtual ~HighwayPatrol() override;
+    HighwayPatrol(const HighwayPatrol&);
+    HighwayPatrol& operator=(const HighwayPatrol&) = default;
+
```

Comment:
move this copy-assign operator implementation to the cpp file, with the other implementations.
- [ ] revise